src/commands/graph/graph.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+const { createGraphEditCommand } = require('./graph-edit')
+const { createGraphPullCommand } = require('./graph-pull')
+
+/**
+ * The graph command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ */
+const graph = (options, command) => {
+  command.help()
+}
+
+/**
+ * Creates the `netlify graph` command
+ * @param {import('../base-command').BaseCommand} program
+ * @returns
+ */
+const createGraphCommand = (program) => {
+  createGraphEditCommand(program)
+  createGraphPullCommand(program)
+
+  return program
+    .command('graph')
+    .description('(Beta) Control the Netlify Graph functions for the current site')
```

Comment:
do we hide this command for now or should it be visible to everybody?

Reply:
This is supposed to be internal only for the first merge.